test/framework/src/main/java/org/opensearch/transport/AbstractSimpleTransportTestCase.java

```diff
@@ -2226,10 +2226,12 @@ public void testHandshakeUpdatesVersion() throws IOException {
                 TransportRequestOptions.Type.STATE
             );
             try (Transport.Connection connection = serviceA.openConnection(node, builder.build())) {
-                // OpenSearch [1.0:2.0) in bwc mode should only "upgrade" to Legacy v7.10.2
+                // OpenSearch [1.0:3.0) in bwc mode should only "upgrade" to Legacy v7.10.2
                 assertEquals(
                     connection.getVersion(),
-                    version.onOrAfter(Version.V_1_0_0) && version.before(Version.V_2_0_0) ? LegacyESVersion.V_7_10_2 : version
+                    version.onOrAfter(Version.V_1_0_0) && version.before(Version.fromId(3000099 ^ Version.MASK))
```

Comment:
Leaving it up to you, may be duplicating and using V_3_0_0 here would make sense (no need to make the constant public):

```
 // @todo remove in 3.0.0
 private static final Version V_3_0_0 = Version.fromId(3000099 ^ Version.MASK);
```

Reply:
That's a good idea.. switched to pkg private and reusing in test.